src/styles/abstracts/_variables.scss

```diff
@@ -0,0 +1,51 @@
+// font file paths
+$circularstd-font-family: 'circularstd' !default;
```

Comment:
it would be cleaner if we grouped all related font family properties together in a map:

```
$circularStd-reglar: (
fontweight: // fontweight goes here, remove the individual variables
path: // path goes here, remove the individual variables
... other properties that the mixin takes, can go in the map
)
```

just pass the map to the mixin.